### ISSUE TITLE
How to use relative path as save path?

### ISSUE THREAD

**Issue by calvindio** [Jaheen/VSCode-Live-Pug-Compiler#2]
`./htmlFolder`
I want to create an html file in the folder named 'htmlFolder' in the current folder where the pug file reside instead of the workspace root
`../folderB`
should create the html file in the folderB which is parallel to the current folder where the pug file reside

**Comment by Jaheen** [comment]
You should change the settings.json file as instructed in readme file.
`json
{
    "livePugCompiler.savePath":"/location to save compiled Html"
}
`
/ refers to root directory of the workspace.
replace the location with "/currentFolder/somefolder".

**Comment by spencermountain** [comment]
thanks for this extension Jaheen.
i was hoping for the same feature:
```
dirA
   - build/foo.html
   - foo.pug
dirB
   - build/bar.html
   - bar.pug
```
where `livePugCompiler.savePath` would be './build' - relative to the file's directory.
cheers!

**Comment by gavin310** [comment]
Yeah there should be a way to compile the HTML into the same folder as the current PUG file without having to specify it. Either way, it's a great extension!

**Comment by cristiulian** [comment]
This is a great extension!
I think I found bug related to the save paths.
My workspace folder structure is something like this:
/
    /dist
        /html
    /src
        /pug
            /index.pug

I've set the output path to ./dist/html and I'm expecting that the index.html file will be created in that folder.
However, the generated file is in ./dist/html/src/pug. Basically the extension creates the /src/pug folders inside /dist/html.

**Comment by cristiulian** [comment]
This is a great extension!
I think I found bug related to the save paths.
My workspace folder structure is something like this:
```/
    /dist
        /html
    /src
        /pug
            /index.pug
```

I've set the output path to ./dist/html and I'm expecting that the index.html file will be created in that folder:
```/
    /dist
        /html/index.html
    /src
        /pug
            /index.pug
```
However, the generated file is in ./dist/html/src/pug. Basically the extension creates the /src/pug folders inside /dist/html:
```/
    /dist
        /html/src/pug/index.html
    /src
        /pug
            /index.pug
```

**Comment by l1ghtsword** [comment]
I'm getting the same issue

Setting = "livePugCompiler.savePath": "/build/html"

Source: /src/html/index.pug
Dest: /build/html/index.html

result = /build/html/src/html/index.html
--------------

For whatever reason it is appending "/src/html" to the build path...

**Comment by rablinz** [comment]
I'm a bit confused about the save path. Each time I start a new project or add a new folder to a workspace, I need to go into Visual Studio Code's global settings and change the global save path to my local project folder? Is that correct?
